core/lib/config/src/configs/database.rs

```diff
@@ -128,7 +128,7 @@ impl DBConfig {
 /// Collection of different database URLs and general PostgreSQL options.
 /// All the entries are optional, since some components may only require a subset of them,
 /// and any component may have overrides.
-#[derive(Debug, Clone, PartialEq)]
+#[derive(Debug, Clone, PartialEq, Serialize, Deserialize)]
```

Comment:
All configs have protobuf serialization, they shouldn't be serde serialized, if they are not already